ocaml/xapi/xapi_vdi_helpers.ml

```diff
@@ -19,10 +19,26 @@ open Client
 open Db_cache_types
 open Pervasiveext
 open Threadext
+open Db_filter_types
```

Comment:
I would prefer if this was a local `let open Db_filter_types in` within the scope of `create_rrd_vdi`, rather than a top-level `open`.